backend/EasyMeets.Core/EasyMeets.Core.Common/Validation/ValidationExtensions.cs

```diff
@@ -91,7 +90,10 @@ public static bool IsValidDateFormat(this DateFormat value)
 
     public static bool IsValidUrl(this string value)
     {
-        var pattern = RegularExpressions["Url"];
-        return Regex.IsMatch(value, pattern);
+        if (value.StartsWith(@"https://easymeetsblobstorage.blob.core.windows.net/fileupload/"))
```

Comment:
It would be better to move this to constants

Reply:
done